### ISSUE TITLE
mypy error - Module 'aiohttp' has no attribute 'ClientTimeout'

### ISSUE THREAD

**Issue by thanosexcite** [aio-libs/aiohttp#3468]
## Long story short

I have the following example Python file:

example.py
```py
from aiohttp import ClientSession
from aiohttp import ClientTimeout

timeout = ClientTimeout()
print(timeout)
```

## Expected behaviour

Mypy returns no errors.

## Actual behaviour

Running mypy on the above example file returns the following error:

```
example.py:2: error: Module 'aiohttp' has no attribute 'ClientTimeout'
```

## Steps to reproduce

```
$ docker run --rm -it -v `pwd`:`pwd` python:3.6.7-slim bash
root@8a1b86a205ee:/# pip install mypy aiohttp
...
Successfully installed aiohttp-3.5.1 async-timeout-3.0.1 attrs-18.2.0 chardet-3.0.4 idna-2.8 idna-ssl-1.1.0 multidict-4.5.2 mypy-0.650 mypy-extensions-0.4.1 typed-ast-1.1.1 typing-extensions-3.6.6 yarl-1.3.0
root@8a1b86a205ee:/# python /home/thanos/Dev/example/example.py 
ClientTimeout(total=None, connect=None, sock_read=None, sock_connect=None)
root@8a1b86a205ee:/# mypy /home/thanos/Dev/example/example.py 
home/thanos/Dev/example/example.py:2: error: Module 'aiohttp' has no attribute 'ClientTimeout'

```

**Comment by asvetlov** [comment]
Looks like `mypy` doesn't respect wildcard imports like
```
from .client import *  # noqa
```

We should get rid of wildcards everywhere.
Looks like trio did it for the same reason.

The fix should be backported to 3.5

**Comment by asvetlov** [comment]
@thanosexcite are you interested in making a PR with fix?

**Comment by webknjaz** [comment]
Other linters like pylint also fail to follow wildcard imports BTW. It's an old known issue.

**Comment by thanosexcite** [comment]
@asvetlov OK, I'll give it a shot. Should I make the PR to master or to the 3.5 branch?

**Comment by asvetlov** [comment]
PR for the master would be enough.
I'll backport it on merging.

**Comment by asvetlov** [comment]
Fixed by #3469
